### PR TITLE
Add API docs

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

**Comment by acrellin** [comment]
@stefanv after this is merged, I'll pick up #148

File: skyportal/handlers/comment.py [hunk 1]
```diff
@@ -42,6 +77,19 @@ def post(self):
...
+            schema: Comment
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+        """
         data = self.get_json()
 
         # TODO: Check ownership
```
Comment (stefanv) [inline skyportal/handlers/comment.py]: Haha, yikes.

File: skyportal/handlers/comment.py [hunk 2]
```diff
@@ -56,6 +104,21 @@ def put(self, comment_id):
...
+            required: true
+            schema:
+              type: integer
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+        """
         # TODO: Check ownership
```
Comment (stefanv) [inline skyportal/handlers/comment.py]: Even more scary? :)

File: Makefile [hunk 3]
```diff
@@ -37,6 +37,9 @@ html: | doc_reqs
 load_demo_data: | dependencies
 	@PYTHONPATH=. python tools/load_demo_data.py
 
+api-docs: | doc_reqs
+	@PYTHONPATH=. python tools/openapi/build-spec.py
```
Comment (stefanv) [inline Makefile]: Where's the step to build the static pages?

**Comment by acrellin** [comment]
@stefanv should be good to go now

**Comment by stefanv** [comment]
Great, this looks good; then we just need an update to `.travis/deploy_docs.sh` as well.

**Comment by acrellin** [comment]
@stefanv what's the update needed to `deploy_docs`?

**Pull request merged**